evap/staff/forms.py

```diff
@@ -31,7 +31,22 @@ class ImportForm(forms.Form):
 
 
 class UserImportForm(forms.Form):
-    excel_file = forms.FileField(label=_("Excel file"))
+    excel_file = forms.FileField(label=_("Import from Excel file"), required=False)
+    course = forms.ModelChoiceField(Course.objects.all(), required=False, label=_("Copy from Course"))
+
+    choices = [('', '---------')]
+    for semester in Semester.objects.all():
```

Comment:
this is evaluated at module import time, and apparently stuff hasn't been initialized completely at that point.